libs/ledger-live-common/src/families/bitcoin/js-getTransactionStatus.ts

```diff
@@ -13,8 +13,9 @@ import type { Account } from "@ledgerhq/types-live";
 import type { Transaction, TransactionStatus } from "./types";
 import { calculateFees, validateRecipient, isTaprootRecipient } from "./cache";
 import { TaprootNotActivated } from "./errors";
-import cryptoFactory from "./wallet-btc/crypto/factory";
+import * as utils from "./wallet-btc/utils";
```

Comment:
import  { computeDustAmount } is not better if you only have this ?

Reply:
@lvndry ^ just in case